src/clj_http_hystrix/core.clj

```diff
@@ -74,14 +77,6 @@
       (warn exception message)
       (warn message))))
 
-(defn status-codes
```

Comment:
If you could leave this in I'll happily merge this. Thanks a lot for the contribution, appreciate the tidying!